pkg/controller/controller.go

```diff
@@ -265,7 +268,6 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 		},
 		CapacityRange: &csi.CapacityRange{
 			RequiredBytes: int64(volSizeBytes),
-			LimitBytes:    int64(volSizeBytes),
```

Comment:
I've also removed `LimitBytes`. Rationale is that some storage plugins may have a min volume size that is larger than `RequiredBytes` and current Kubernetes behavior is to allow the greater sized volume to be created and bound to the requested PVC. This will align the behavior with existing Kubernetes behavior. Additionally, very few storage systems are able to give back a volume with the *exact* amount of bytes requested, it might have to round up a little (or a lot), and we should not fail in those use cases.